timeseries/setup.py

```diff
@@ -26,7 +26,7 @@
     "joblib~=1.1",
     "numpy",
     "scipy",
-    "pandas",
+    "pandas>=1.4.0",
```

Comment:
Please consider updating `core/_setup_utils.py` instead.

```
'pandas': '>=1.2.5,!=1.4.0,<1.6',
```
is the current range.

Change to:

```
'pandas': '>1.4.0,<1.6',
```

Reply:
once this is done and the current LoC change is reverted, I approve, feel free to merge without syncing with me